### ISSUE TITLE
[Spike] Fixes discovery and planning

### ISSUE THREAD

**Issue by lisacapaccioli** [department-of-veterans-affairs/va.gov-team#89739]
As a Product Manager, I need to understand the issues for the Veteran evidence document uploads that are preventing the documents from actually "uploading" in the VA systems, so that we can refine and prioritize the work on them. 

**For this to be successful**
1. Create a report identifying all the issues 
2. Identify the issue (using what was done in discovery)
3. Identify the root cause (example - it it a backend problem that a Veteran has no control over or is it something that the Veteran has to change because we have rules for evidence uploads)
4. If possible, identify the suggested solution (not all issues can be solved so easily, but some might, provide some ideas that the Team could discuss)
5. If possible, identify the effort or additional work that is needed (does it need more discovery, is there a dependency, is there UI changes necessary, etc.)
6. If any, identify risks (excluding the known risk which is that a Veteran may not get their benefits) 
7. Sharable - this will be shared in the review and to our partners

**Suggestions for the report**

- [Store the report in the Github team sensitive repository ](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/tree/master/teams/benefits)
- Organize the work so that it's easy to follow (e.g., groups, categories, whatever floats your boat) 
- Know your audience (write not just for engineers, but for the whole team and our partners)
- Get feedback (Ask team members - PM, Eng, UX - to test your report and see if it makes sense before you write the whole thing and share it out. You don't want to find out too late that it's confusing and too hard to follow.)
- Keep in mind the purpose of the report - we need to prioritize the work, and too little information may not help, too much may make it complicated. 
- Add any details that you think will help with the purpose (share your thoughts, explore ideas)

Resources: 

https://developer.va.gov/explore/api/benefits-intake/docs?version=current
https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1718308823134/02c2c9ea74f16b276692af8f31eb65202acc7928?sender=ua8d1f105fba69675aba15464
https://github.com/department-of-veterans-affairs/va.gov-team/issues/88577#issuecomment-2240489981

**Comment by ajones446** [comment]
High-level suggestions:
1. Refine backlog of existing tickets, several of these issues are already open and should be condensed/closed in coordination with other teams following them.
2. Determine a strategy for monitoring ongoing error numbers as fixes are introduced.
3. Look into adding `.txt` file validations (like ValidatePdf module) to enforce encoding and verify mime types. 

Detailed notes reported [here.](https://docs.google.com/spreadsheets/d/1s6r41HMZH2K_ti8TWJG61Vgi2j_dwdgYPLgIXc2JoQw/edit?usp=sharing)

**Comment by ajones446** [comment]
Going forward, the upload process will use the Lighthouse Benefits Documents API instead of EVSS as part of the [Lighthouse Document Upload Migration Epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/83837). The Lighthouse API has additional requirements that need to be validated, which will be explored in the following [spike for evidence uploads error responses on Lighthouse.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/91097)


The root cause of many existing EVSS errors are likely to cause an error on Lighthouse as well, specifically the following issues:
 - PDF dimensions too large:  #77041 
 - Mismatched Mimetype: #81480 
 - Incorrect Encoding: #78251


These following tickets are related to EVSS upload failures and may be applicable to Lighthouse, listing here for reference:
   - EVSS_15001 catch-all bucket of potential PDF errors: #81481
   - EVSS_15006 generic corrupted PDF error: #60165
   - "File Upload" error - potential BIRLS ID issue #74381
